db/system_keyspace.cc

```diff
@@ -72,14 +72,9 @@ namespace {
         static const std::unordered_set<sstring> extra_durable_tables = {
             system_keyspace::PAXOS,
             system_keyspace::SCYLLA_LOCAL,
-            system_keyspace::RAFT,
-            system_keyspace::RAFT_SNAPSHOTS,
-            system_keyspace::RAFT_SNAPSHOT_CONFIG,
-            system_keyspace::DISCOVERY,
             system_keyspace::BROADCAST_KV_STORE,
             system_keyspace::TOPOLOGY,
-            system_keyspace::CDC_GENERATIONS_V3,
-            system_keyspace::TABLETS,
+            system_keyspace::CDC_GENERATIONS_V3
```

Comment:
nit: we usually leave ',' after the last element to make patches extending the table less noisy.

Reply:
fixed